generate.py

```diff
@@ -70,30 +72,32 @@ def generate(
 
 
 def main(
-    prompt: str = "Hello, my name is",
+    prompt: str = DEFAULT_PROMPT,
     *,
     num_samples: int = 1,
     max_new_tokens: int = 50,
     top_k: int = 200,
     temperature: float = 0.8,
-    checkpoint_dir: Path = Path(f"checkpoints/stabilityai/stablelm-base-alpha-3b"),
+    checkpoint_dir: Path = Path(DEFAULT_CP),
     quantize: Optional[str] = None,
+    seed: Optional[int] = 1234
```

Comment:
Can you rebase this PR so that you only add the `seed` argument? Thank you!

Reply:
sure.